STAR/draft-ietf-acme-star.md

```diff
@@ -516,6 +516,72 @@ setting it to true.
 
 If the server accepts the request, it MUST reflect the key in the Order.
 
+## Computing notBefore and notAfter of STAR Certificates
+
+We define the "nominal renewal date" as the point in time when a new short-term
+certificate for a given STAR Order is due.  It is a multiple of the Order's
+recurrent-certificate-validity which starts with the issuance of the
+first short-term certificate and it is upper bounded by the Order's
+recurrent-end-date ({{fignrd}}).
+
+~~~
+    rcv    - STAR Order's recurrent-certificate-validity
+    red    - STAR Order's recurrent-end-date
+    nrd[i] - nominal renewal date of the i-th STAR certificate
+
+
+                 .-rcv-.   .-rcv-.   .-rcv-.   ._rcv_.
+                /       \ /       \ /       \ /  red  \
+    -----------o---------o---------o---------o----X-------> t
+              nrd[0]    nrd[1]    nrd[2]    nrd[3]
+~~~
+{: #fignrd title="Nominal Renewal Date"}
+
+The rules to determine the notBefore and notAfter values of the i-th STAR
+certificate are as follows:
+
+~~~
+  notBefore = nrd[i] - predating
+  notAfter = min(nrd[i] + rcv, red)
+~~~
+
+where "predating" is the max between the (optional)
+recurrent-certificate-predate (rcp) and the amount of pre-dating that the
+server needs to add to make sure that all certificates being published are
+valid at the time of publication ({{fetching-certificates}}).  The server
+pre-dating is a fraction of rcv (1/n with n>=2).
```

Comment:
I think the sense of comparison may be wrong here, because the new cert must be available _at the latest_ halfway through the lifetime of the previous one. Maybe should simply be `rcv/2`.

Reply:
We say "at the latest halfway through the lifetime of the currently active certificate", so it has to be rcv/2 or smaller.  I've changed the parenthetical to maybe be clearer.

Reply:
No, it's rcv/2 or larger in fact, because rcv/2 (actually, the value "predating" that depends on it) is *subtracted* from the nrd.

Reply:
🤦‍♂️

Reply:
thanks, should be fixed in e8a24c2